app/src/main/java/edu/usf/sas/pal/muser/model/DeviceInfo.kt

```diff
@@ -0,0 +1,30 @@
+package edu.usf.sas.pal.muser.model
```

Comment:
Same - if this is derived from OBA, could you please add license and link to that class?